README.md

```diff
@@ -22,22 +22,22 @@ This library is more suited to ETL than interactive queries, since large amounts
 
 ## Installation
 
-This library requires Apache Spark 1.4+ and Amazon Redshift 1.0.963+.
+This library requires Apache Spark 2.0+ and Amazon Redshift 1.0.963+.
```

Comment:
Created a new branch at https://github.com/databricks/spark-redshift/tree/branch-1.x for holding the 1.x code.